WooCommerce/WooCommerceTests/ViewRelated/Orders/Order Details/Addresses/EditAddressFormViewModelTests.swift

```diff
@@ -69,4 +86,18 @@ private extension EditAddressFormViewModelTests {
                        phone: "333-333-3333",
                        email: "scrambled@scrambled.com")
     }
+
+    func sampleAddressWithEmptyNullableFields() -> Address {
+        return Address(firstName: "Johnny",
+                       lastName: "Appleseed",
+                       company: "",
+                       address1: "234 70th Street",
+                       address2: "",
+                       city: "Niagara Falls",
+                       state: "NY",
+                       postcode: "14304",
+                       country: "US",
+                       phone: "",
+                       email: "")
+    }
```

Comment:
4 optional fields are empty strings here. `EditAddressFormViewModel` will serialize them into `nil` values in `addressFromFields` computed var.